docs/sources/_index.md

```diff
@@ -19,11 +19,36 @@ keywords:
 Beyla is currently in [public preview](/docs/release-life-cycle/). Grafana Labs offers limited support, and breaking changes might occur prior to the feature being made generally available.
 {{% /admonition %}}
 
-Grafana Beyla is an eBPF based application auto-instrumentation tool to track total request time.
+## Effortless application auto-instrumentation for Application Observability
+
+Instrumenting an application to get metrics and traces typically requires adding a language agent to your application deployment/packages. In some compiled languages like Go or Rust, you have to manually add tracepoints into your code. In both cases, you need to redeploy the instrumented version of your application to your staging/production servers.
+
+Grafana Beyla is an eBPF based application auto-instrumentation tool that lets you easily get started with Application Observability. We leverage eBPF to automatically inspect your application executables and the OS networking layer, to capture basic trace spans about your web transactions, as well as Rate-Errors-Duration (RED) metrics for your Linux HTTP/S and gRPC services. 
+All of the data capture is done without any modifications to your application code or configuration.
+
+### Couple of reasons why to use Beyla
+
+- Beyla is vendor agnostic. You can export the generated metrics and traces in OpenTelemetry format, as
+well as native Prometheus metrics.
+- Beyla can auto-instrument applications written in various programming languages, for example: 
+Go, C/C++, Rust, Python, Ruby, Java (including GraalVM Native), NodeJS, .NET and others.
+- Beyla is efficient. The instrumentation and the data capture is done with natively compiled code,
+even if you are instrumenting applications written in interpreted languages like Python. Because of this,
+Beyla provides a way to capture application metrics and traces, with much lesser overhead than using a
+language specific instrumentation SDK.
+- Beyla is Kubernetes ready. You can run Beyla in any Linux environment, but if you run in Kubernetes, Beyla 
+can listen to your Kubernetes API to decorate metrics and traces with Pods and Services metadata. 
+- Beyla integrates with the Grafana Agent, so if you are already a Grafana customer, you can easily
+get started.
+
+
+## Request time instead of service time, for your server-side application code
```

Comment:
I'd actually move this to another document, and link it from here as one of the features of Beyla.

Reply:
Sounds good, I'll make that change.